packages/blockchain-wallet-v4-frontend/src/data/components/buySell/sagaRegister.ts

```diff
@@ -86,5 +86,15 @@ export default ({ api, coreSagas, networks }) => {
         yield cancel(pollTask)
       }
     )
+
+    yield takeLatest(
+      actions.startPollBuyQuote.type,
+      function* (payload: ReturnType<typeof actions.startPollBuyQuote>) {
+        if (pollTask && pollTask.isRunning()) yield cancel(pollTask)
```

Comment:
we probably want to use a different pollTask than the Sell one above. Lets make two variables
`buyPollTask` and `sellPollTask` and then update everything accordingly